client/object_get.go

```diff
@@ -35,10 +35,10 @@ type prmObjectRead struct {
 	bearer    token.BearerToken
 
 	cnrSet bool
-	cnr    cid.ID
+	cnrID  cid.ID
```

Comment:
Why?

Reply:
when i first time refactored (fixed after refactor) that code I tried to convert it to a container from v2, but then I realized that it is not a container

so at least we have one case of misunderstanding

Reply:
Ok, nvm, hope u'll do it more carefully next time.